fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -15,7 +15,7 @@
  */
 package org.fcrepo.auth.common;
 
-import static org.fcrepo.kernel.api.FedoraJcrTypes.JCR_CONTENT;
+import static org.modeshape.jcr.api.JcrConstants.JCR_CONTENT;
```

Comment:
`find . -name 'pom.xml' -exec grep -H "fcrepo-kernel-modeshape" "{}" \;`
results in:
```
./fcrepo-kernel-modeshape/pom.xml:  <artifactId>fcrepo-kernel-modeshape</artifactId>
./fcrepo-connector-file/pom.xml:      <artifactId>fcrepo-kernel-modeshape</artifactId>
./fcrepo-auth-common/pom.xml:      <artifactId>fcrepo-kernel-modeshape</artifactId>
./fcrepo-http-api/pom.xml:      <artifactId>fcrepo-kernel-modeshape</artifactId>
./fcrepo-http-api/pom.xml:      <artifactId>fcrepo-kernel-modeshape</artifactId>
./pom.xml:    <module>fcrepo-kernel-modeshape</module>
./fcrepo-webapp/pom.xml:      <artifactId>fcrepo-kernel-modeshape</artifactId>
./fcrepo-webapp/target/console-work/META-INF/maven/org.fcrepo/fcrepo-webapp/pom.xml:      <artifactId>fcrepo-kernel-modeshape</artifactId>
./fcrepo-http-commons/pom.xml:      <artifactId>fcrepo-kernel-modeshape</artifactId>
./fcrepo-boms/fcrepo4-bom/pom.xml:        <artifactId>fcrepo-kernel-modeshape</artifactId>
./fcrepo-jms/pom.xml:      <artifactId>fcrepo-kernel-modeshape</artifactId>
```